### PR TITLE
CNS-341 create cookbook plans

### PR DESCRIPTION
**Pull request by orenl-lava**

### CODE REVIEW

File: x/pairing/keeper/unresponsive_provider.go [hunk 1]
```diff
@@ -29,14 +29,14 @@ func (k Keeper) UnstakeUnresponsiveProviders(ctx sdk.Context, epochsNumToCheckCU
...
-		epochTemp = previousEpoch
+	// To check for punishment, we have to go back:
+	//   recommendedEpochNumToCollectPayment+
+	//   max(epochsNumToCheckCUForComplainers,epochsNumToCheckCUForUnresponsiveProvider)
+	// epochs from the current epoch.
+	minHistoryBlock := k.getBlockEpochsAgo(ctx, currentEpoch, largerEpochsNumConst+recommendedEpochNumToCollectPayment)
+	if minHistoryBlock == 0 {
+		// not enough history, do nothing
+		return nil
 	}
```
Comment (SeanZoR) [inline x/pairing/keeper/unresponsive_provider.go:40]: @orenl-lava are these changes suppose to be part of this PR?

File: cookbook/plans/adventurer.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "proposal": {
+        "title": "Add default plan proposal",
+        "description": "A proposal of a default plan",
```
Comment (SeanZoR) [inline cookbook/plans/adventurer.json:4]:
```suggestion
        "title": "Add testnet plan proposal - adventurer",
        "description": "A proposal for new plan",

```

File: cookbook/plans/explorer.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "proposal": {
+        "title": "Add default plan proposal",
+        "description": "A proposal of a default plan",
```
Comment (SeanZoR) [inline cookbook/plans/explorer.json:4]:
```suggestion
        "title": "Add testnet plan proposal - explorer",
        "description": "A proposal for new plan",

```

File: cookbook/plans/whale.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "proposal": {
+        "title": "Add default plan proposal",
+        "description": "A proposal of a default plan",
```
Comment (SeanZoR) [inline cookbook/plans/whale.json:4]:
```suggestion
        "title": "Add testnet plan proposal - whale",
        "description": "A proposal for new plan",

```

**Review by SeanZoR — COMMENTED** [review]
Plans look good. See my comments/suggestions

File: x/pairing/keeper/unresponsive_provider.go (hunk 1 above)
Comment (orenl-lava) [reply x/pairing/keeper/unresponsive_provider.go:40]: ugh. good catch.

**Review by oren-lava — APPROVED** [review]

**Review by omerlavanet — APPROVED** [review]

**Pull request merged**